wpengine/faustjs
# Improve toolbar accessibility

## Tasks


## Description

These changes improve the toolbar accessibility by allowing menus to be opened with the "enter" key. Once the menu is opened, a user should be able to tab through the menu items.

## Testing

1. Checkout branch: `MERL-941-toolbar-a11y`
2. `npm run build -w @faustwp/core`
3. `npm run dev -w @faustwp/getting-started-example`
4. Go to your connected WordPress site and preview a post to view/authenticate in Faust.
5. Tab over to the "My Account" dropdown on the right side of the Toolbar.
6. Observe that menu opens when "enter" or "space" is pressed, and you can tab through the menu while it's open.
